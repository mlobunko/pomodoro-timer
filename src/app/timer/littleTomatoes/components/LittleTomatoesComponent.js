import React from "react";

import LittleTomato from "./LittleTomatoContainer";
import { StyledLittleTomatoesComponent } from "./styles";

export const LittleTomatoesComponent = () => (
  <StyledLittleTomatoesComponent>
    <LittleTomato order={0} />
    <LittleTomato order={1} />
    <LittleTomato order={2} />
    <LittleTomato order={3} />
  </StyledLittleTomatoesComponent>
);

LittleTomatoesComponent.displayName = "LittleTomatoes";

export default LittleTomatoesComponent;
