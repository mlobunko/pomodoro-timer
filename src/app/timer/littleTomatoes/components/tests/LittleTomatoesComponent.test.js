import React from "react";
import { shallow } from "enzyme";

import { LittleTomatoesComponent } from "../LittleTomatoesComponent";

describe("render", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<LittleTomatoesComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
