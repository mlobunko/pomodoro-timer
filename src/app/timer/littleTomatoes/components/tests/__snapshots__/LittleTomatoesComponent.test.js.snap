// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<StyledLittleTomatoesComponent>
  <Connect(LittleTomato)
    order={0}
  />
  <Connect(LittleTomato)
    order={1}
  />
  <Connect(LittleTomato)
    order={2}
  />
  <Connect(LittleTomato)
    order={3}
  />
</StyledLittleTomatoesComponent>
`;
